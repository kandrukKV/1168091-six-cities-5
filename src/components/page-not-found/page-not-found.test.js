import React from "react";
import renderer from "react-test-renderer";
import PageNotFound from "./page-not-found";

it(`OffersContentEmpty rendered correctly`, () => {
  const tree = renderer.create(
      <PageNotFound/>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
