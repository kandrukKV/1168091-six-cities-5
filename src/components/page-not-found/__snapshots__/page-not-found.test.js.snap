// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OffersContentEmpty rendered correctly 1`] = `
<h1>
  Page not found or lost connection
</h1>
`;
